examples/build_system/make/rules.mk

```diff
@@ -90,7 +96,7 @@ linkermap: $(BUILD)/$(PROJECT).elf
 
 # --------------- Jlink -----------------
 ifeq ($(OS),Windows_NT)
-  JLINKEXE = JLink.exe
+  JLINKEXE = C:\Program Files\SEGGER\JLink\JLink.exe
```

Comment:
This should be undone, since using a static directory is not recomended

Reply:
It would be desirable to undo this change, but test it with Nicole and Fausto first.